exchanges/credentials.go

```diff
@@ -98,10 +98,10 @@ func (b *Base) AreCredentialsValid(ctx context.Context) bool {
 
 // GetDefaultCredentials returns the exchange.Base api credentials loaded by
 // config.json
-func (b *Base) GetDefaultCredentials() *account.Credentials {
+func (b *Base) GetDefaultCredentials() *accounts.Credentials {
 	b.API.credMu.RLock()
 	defer b.API.credMu.RUnlock()
-	if b.API.credentials == (account.Credentials{}) {
+	if b.API.credentials == (accounts.Credentials{}) {
```

Comment:
suggestion we might be able to just rm this? or just check for key instead of allocating an empty struct.

Reply:
I didn't sniff the credentials section too hard, and I'm not touching it now, because I know it'll unravel into a lot of changes. This PR is already way overdue and merge hell.